src/components/Header/Header.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  title: string,
+  handleSubmit: (event: React.FormEvent<HTMLFormElement>) => void,
+  createNewTitle: (title: string) => void,
+  completedTodos: Todo[],
+  activeTodos: Todo[],
+  updateTodos: (todoId: number[], data: boolean) => void,
+  isDisableInput: boolean,
+};
+
+export const Header: React.FC<Props> = React.memo(
+  ({
+    title,
+    handleSubmit,
+    createNewTitle,
+    completedTodos,
+    activeTodos,
+    updateTodos,
+    isDisableInput,
+  }) => {
+    return (
+      <header className="todoapp__header">
+        <button
+          aria-label="btn"
+          type="button"
```

Comment:
Consider providing a more descriptive `aria-label` for the button, such as 'Toggle All Todos' to improve accessibility.